docs/_snippets/framework/extending-content-arbitrary-attribute-values.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals ClassicEditor, console, window, document */
+
+import { CS_CONFIG } from '@ckeditor/ckeditor5-cloud-services/tests/_utils/cloud-services-config';
+import Font from '@ckeditor/ckeditor5-font/src/font';
```

Comment:
Can't add a plugin to a build:

![image](https://user-images.githubusercontent.com/156149/61625169-627f2c80-ac7a-11e9-89b7-90d17a3bf66e.png)